### PR TITLE
fix: next_event py signature using int instead of timedelta

### PR DESCRIPTION
**Pull request by itayd**
same as sleep

### CODE REVIEW

File: runtimes/pythonrt/py-sdk/autokitteh/events.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def unsubscribe(subscription_id: str) -> None:
     pass
 
 
-def next_event(subscription_id: str, *, timeout: timedelta = None) -> AttrDict:
```
Comment (daabr) [inline runtimes/pythonrt/py-sdk/autokitteh/events.py:21]: All of our docs and samples refer to using a `timedelta` and not an `int`. The main reason is that `int` doesn't have a clear meaning - is it seconds? milliseconds? I was going to fix it in the other direction after #790 anyway.

**Review by tebeka — CHANGES_REQUESTED** [review]
I think timeout should be timedelta. See https://github.com/autokitteh/autokitteh/pull/797

File: runtimes/pythonrt/py-sdk/autokitteh/events.py (hunk 1 above)
Comment (daabr) [reply runtimes/pythonrt/py-sdk/autokitteh/events.py:21]: See https://github.com/autokitteh/autokitteh/pull/797

Comment (itayd) [reply runtimes/pythonrt/py-sdk/autokitteh/events.py:21]: @tebeka @daabr i get it, but isn't it a bit tedious to use? sleep is using int. up to you.

**Pull request closed without merging**